crates/re_types/definitions/rerun/archetypes/scalar.fbs

```diff
@@ -0,0 +1,36 @@
+include "fbs/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/components.fbs";
+
+namespace rerun.archetypes;
+
+// ---
+
+/// Log a double-precision scalar.
+///
+/// The current simulation time will be used for the time/X-axis, hence scalars
+/// cannot be timeless!
+///
```

Comment:
I think it would make sense to refer to the "related" archetypes of `series_line` and `series_point`. Since it confused users in the past I think it's also important to point out that we expect you to log the same ones on the same path